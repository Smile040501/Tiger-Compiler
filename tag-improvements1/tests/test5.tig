x := -1;

for x := 1 to 10 by 2 do
    println x;
done;

for x := 10 to 1 by -3 do
    println x;
done;

println x;
