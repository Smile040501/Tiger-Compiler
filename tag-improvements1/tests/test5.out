1
3
5
7
9
10
7
4
1
-1
